### ISSUE TITLE
Exception handling review in CRAB client

### ISSUE THREAD

**Issue by cinquo** [dmwm/CRABClient#1173]
# From Simon comment:

The point is that the exceptions should in the majority of cases come from the server (e.g. if you can talk to the server and get a response any exception will be an HTTP error from the server). I don't think you need to wrap that further. Also, exceptions aren't that useful in the client - as you say, if one is raised it's basically going to print a message and exit, there's not going to be much beyond that in terms of exception handling - so having a bunch of specific exceptions is somewhat code bloat.
# Basically I'd see the first version of the client have no exceptions (e.g. just use ones from pythons standard library. ValueError?, HTTPError etc.) and only add specific exceptions in very complicated, rare cases. I wouldn't be surprised if the final client had no specialised exceptions in either...
